pkg/log_urls/log_urls_test.go

```diff
@@ -23,17 +23,29 @@ func TestGenerateLogURLs(t *testing.T) {
ed:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:nginx-container),type:phrase),query:(match_phrase:(kubernetes.container_name:nginx-container)))),interval:auto,query:(language:kuery,query:''),sort:!())", "https://localhost:5601/app/discover#/?_a=(columns:!(),dataSource:(dataViewId:'035bffd1-b4b5-4123-af57-d89368cabfd8',type:dataView),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:_index,negate:!f,params:(query:fluentd),type:phrase),query:(match_phrase:(_index:fluentd))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.pod_id,negate:!f,params:(query:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'),type:phrase),query:(match_phrase:(kubernetes.pod_id:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:busybox-container),type:phrase),query:(match_phrase:(kubernetes.container_name:busybox-container)))),interval:auto,query:(language:kuery,query:''),sort:!())", "https://localhost:5601/app/discover#/?_a=(columns:!(),dataSource:(dataViewId:'035bffd1-b4b5-4123-af57-d89368cabfd8',type:dataView),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:_index,negate:!f,params:(query:fluentd),type:phrase),query:(match_phrase:(_index:fluentd))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.pod_id,negate:!f,params:(query:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'),type:phrase),query:(match_phrase:(kubernetes.pod_id:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:echo-3),type:phrase),query:(match_phrase:(kubernetes.container_name:echo-3)))),interval:auto,query:(language:kuery,query:''),sort:!())", "https://localhost:5601/app/discover#/?_a=(columns:!(),dataSource:(dataViewId:'035bffd1-b4b5-4123-af57-d89368cabfd8',type:dataView),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:_index,negate:!f,params:(query:fluentd),type:phrase),query:(match_phrase:(_index:fluentd))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.pod_id,negate:!f,params:(query:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'),type:phrase),query:(match_phrase:(kubernetes.pod_id:'a8c9b834-bc4a-4438-8841-23a5ffb9164c'))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'035bffd1-b4b5-4123-af57-d89368cabfd8',key:kubernetes.container_name,negate:!f,params:(query:echo-4),type:phrase),query:(match_phrase:(kubernetes.container_name:echo-4)))),interval:auto,query:(language:kuery,query:''),sort:!())"},
```

Comment:
Any chance to have this line separated in several so that it's a little more readable? 
If it could be done I'd suggest to add the same approach for the rest of the `expected` parameters but this is specially hard to follow given the length.

Reply:
Yeah, I hated this too.

I've moved the expected output to files, which I think should make things much better.

Reply:
yes! I think it looks much better now :+1: